src/docs/binding-variables.md

```diff
@@ -0,0 +1,31 @@
+# Binding Variables
+
+Variables in Swift come in two general forms: computed and stored.  Unlike C#, Swift allows variables to exist at the top level of a module being effectively global.
+
+With enable-module-evolution set, all `let` variables will have a getter. Variables that are declared with `var` will have a getter as well as a setter. It is an error to make a set-only variable.
+
+Variables can be:
+- global
+- static
+- instance
+- class (computed only)
+- mutating set (value types only)
+- open (instance in classes only)
+
+## Language Parity Mismatches
+- Naming - Swift can have variable names that are unpronouncable in C#
+- Class - the notion of class variables doesn't exist in C#
```

Comment:
Similar question to my question about class functions earlier, what about static properties? How do they differ?